platform/AppStoreConnect/Tests/AppStoreConnectTests/Base64p8KeyParsingTests.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import XCTest
+@testable import AppStoreConnect
+
+class Base64p8KeyParsingTests: XCTestCase {
+    func test_properlyExtractsTheKey() throws {
+        // The actual key
+        let key = """
```

Comment:
Same as "repeatedRequests" above, possibly in a configuration file at a later stage.